lua/lint/linters/cue.lua

```diff
@@ -0,0 +1,38 @@
+return {
+  cmd = "cue",
+  args = { "vet" },
+  stdin = false,
+  append_fname = true,
+  ignore_exitcode = true,
+  stream = "stderr",
+  parser = function(output)
+    if output == "" then
+      return {}
+    end
+
+    local diagnostics = {}
+    local lnum, col, message
+    local pattern = ":(%d+):(%d+)$"
+
+    for _, line in ipairs(vim.fn.split(output, "\n")) do
+      if line:match("^%s+") then
+        lnum, col = line:match(pattern)
+      else
+        message = line:gsub(":", "")
+      end
```

Comment:
Could the parser use `from_errorformat` or `from_pattern` instead - looks like it can be declared with a pattern?

Reply:
Can I? 
The output from `cue vet` spans over 2 lines where the message is on the first line and then the location follows on the next line.
Like so:
```
❯ cue vet example.cue
square bracket must have exactly one element:
    ./example.cue:19:14
string literal not terminated:
    ./example.cue:25:34
```
The readme says 
> The regular expression pattern applied on each line of the output